### DISCUSSION TITLE
How can I set two SV on ModBus?

### DISCUSSION THREAD

**Discussion by tanxoo** [artisan-roaster-scope/artisan#716] [Q&A] [answered]
![pic](https://user-images.githubusercontent.com/39583658/139478034-9b526fc5-6e10-40ea-a65c-4889292b21eb.png)

I have 2 slave on the modbus and I want to controll them on the same time. But I can only fInd way to control one of them by write the slave number on PID block under. How can I controll another slave?

The slave is DTA4848

**Comment by MAKOMO** [comment, marked as answer]
Not sure what you mean by control, but Artisan is only supporting one PID loop (either external or the internal Software PID). However, you can send any MODBUS command, also one that modifies the SV in any of your connected devices, via a custom event button or slider. Anything beyond this is rather special and it might be better to implement your own control software.

**Reply by tanxoo** [reply]
thank you! I add a  slider to  send MODBUS command write(1,18177,{}) ,that I can set two SV together 
But I find there might a bug. the SV is the controlled ,and I can use manual to change SV anytime before. But with the new 
 silder added, the manual can't change zhe SV ,but  R/S or PID can. And after R/S or PID worked , the manual worked again.....
![pic2](https://user-images.githubusercontent.com/39583658/139496703-6f9438ec-50ef-452c-999d-6d1e36a8849b.png)
![pic3](https://user-images.githubusercontent.com/39583658/139496707-535ee2c5-672f-4f74-8aa1-0645eeecb7e1.png)
![pic4](https://user-images.githubusercontent.com/39583658/139496709-2b19ca47-da55-46ed-82f5-6a35dbfe87fc.png)

**Comment by MAKOMO** [comment]
The SV slider is only sending its signal to the connected PID if the PID mode is active as far as I remember.

**Reply by tanxoo** [reply]
forgot it. I set an  upper limit of 210C, that make it didn't work....

**Reply by tanxoo** [reply]
thank you!
